features/parameter-types/rooms.js

```diff
@@ -59,8 +75,25 @@ class PublicityLevel {
 }
 
 defineParameterType({
-  name: "roomKey",
-  regexp: concatRegExp(FLEXIBLE_ARTICLE_ADJECTIVES, /(correct |wrong )?Room Key/)
+  name: "accessCode",
+  regexp: concatRegExp(FLEXIBLE_ARTICLE_ADJECTIVES, /(correct |valid |wrong |empty )?Access Code/),
```

Comment:
add a comment explaining why both `correct` and `valid` must be handled in this file. Is this just to accommodate variations of cucumber strings and to not break existing tests?

Reply:
Good point!

Reply:
Great point! Yes, this is to make it a bit easier to use in steps where either `correct` or `valid` would feel funky

Reply:
Just realized valid and correct is two different concept.

Reply:
They are distinct concepts; however in this particular case they are synonyms. I don't know whether exposing the synonyms is clarifying or confusing.